### DISCUSSION TITLE
🔗 🚀 Feature Request: Add Multi-language support for the Taxonomy Filter (Related to #23418)

### DISCUSSION THREAD

**Discussion by innoforce-dev** [elementor/elementor#23361] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

We have a multi-language site, using Polylang. We'd like to use loop grid to display entries of a custom post type "products". All good, the language specific products can be filtered with the include or exclude properities in the query. 

Using the taxonomy filter, the page shows tax items of all languages. 

### Describe the solution you'd like

Either
* the taxonomy widget only shows the taxo. items of all query-matching items
* or the taxonomy widget will have an option to filter by language

**Comment by nicholaszein** [comment]
Hello @innoforce-dev,

Thanks for contributing your suggestion! 🙏

Your ideas and input as a Web Creator are valuable to us. We're always keen to explore ways to enhance Elementor to meet our community's needs better.

### 💡 Due to the high volume of feature requests we receive and our commitment to continuous improvement, we can't guarantee that every suggestion will be immediately included in our roadmap.

That being said, your idea is important to us. We'll keep your suggestion on our radar, and if it attracts a significant number of votes in the future, we'll consider implementing it.

We appreciate your patience as we take the time to review your request thoroughly. We aim to ensure all decisions align with our mission to provide Web Creators with the tools they need to build their dream websites.

Kind regards

**Comment by nicholaszein** [comment]
Related to:
- #23418

**Comment by Alvaroleguizamon** [comment]
i have the same issue... any news?
